org.supertuxproject.SuperTux.json

```diff
@@ -14,7 +14,8 @@
         "--share=ipc",
         "--socket=pulseaudio",
         "--share=network",
-        "--device=all"
+        "--device=all",
+        "--env=SDL_VIDEODRIVER=wayland,x11"
```

Comment:
What is the point of this? Wayland should be prioritised without this by virtue of this using `socket=fallback-x11` instead of `socket=x11`.

Reply:
Strange, can't reproduce on my system (I just removed the environment variable override using Flatseal, didn't rebuild). I'm using GNOME Wayland. What is your environment?